### ISSUE TITLE
Bully Scholarship Edition - FMVs do not play properly on Windows

### ISSUE THREAD

**Issue by RecklessGlue540** [doitsujin/dxvk#4062]
So, just as you boot up the game with DXVK on Windows, you'll notice that only the audio of the FMVs will play, the video part will just not initialize. Only solution is really skipping them.
It's also worth mentioning that the game on PC and Xbox 360 uses Windows Media Player with a masked interface to play these videos, which in turn are also formatted in an .wmv container. The Wii version is a lot better in this regard since it uses Bink instead, but let's not delve too much from the topic.

Video demonstrating the problem (turn audio on):

https://github.com/doitsujin/dxvk/assets/143945383/dbea7e10-b055-4b05-93d8-c9f4f5ff5f28

### Software information
Bully: Scholarship Edition, version 1.200 + SilentPatch BETA 3 as it fixes major issues.

### System information
- GPU: AMD Radeon RX 6700 10gb
- Driver: 24.5.1
- Windows version: Windows 11 23H2, OS Build: 22631.3672
- DXVK version: 2.3.1

### Apitrace file(s)
Not sure how useful it may be in this case, but here it is anyway:
https://www.mediafire.com/file/ddf9eq6qxdwyfu1/Bully.7z/file

### Log files
[Bully_d3d9.log](https://github.com/user-attachments/files/15722941/Bully_d3d9.log)

**Comment by Blisto91** [comment]
Do you know if older versions of dxvk exhibit the same issue?

**Comment by RecklessGlue540** [comment]
> Do you know if older versions of dxvk exhibit the same issue?

Yeah, tested just now starting from 2.3.1 all the way back to 1.9.4, seems like it's forever been an issue..

**Comment by mrdeathjr28** [comment]
In my case showa white screen with wine vanilla and staging with lastest dxvk but back to cinematics issue, wmv is higly problematic format on wine and seems similar in this case but using this:

![magicwn](https://github.com/doitsujin/dxvk/assets/6601480/9bd7fce5-ec97-48e9-9cd5-7bfbf892d9f3)

show this (maybe can solve this issue, however this problem have more relation with wine than dxvk in my experience with various cinematics formats):

![bla](https://github.com/doitsujin/dxvk/assets/6601480/9ceb2835-4f0a-4ec9-9a2d-714735003476)

![blc](https://github.com/doitsujin/dxvk/assets/6601480/c62c2247-9aab-489b-b174-ba1d5fbfd33e)

![bld](https://github.com/doitsujin/dxvk/assets/6601480/7e5de846-c04b-4874-9c88-6609cf1fd22c)

**Comment by RecklessGlue540** [comment]
So your idea is to re-encode the videos into something better? In this case, that is not possible as the game cannot take literally anything else than .wmv.
That would only be possible if someone were to reverse the FMV decoder from the game and replace it with Bink or something else. Bink is the most preferable imo, although it'd probably require a license lol... like many others actually?

**Comment by mrdeathjr28** [comment]
> So your idea is to re-encode the videos into something better? In this case, that is not possible as the game cannot take literally anything else than .wmv. That would only be possible if someone were to reverse the FMV decoder from the game and replace it with Bink or something else. Bink is the most preferable imo, although it'd probably require a license lol... like many others actually?

curiously in your case stay on windows and then media foundation aka mf must be work (wmv must be work ok on windows), on wine media foundation implementation still is a wip (specially on wmv)

sadly for now i said maybe can be a option for now, however i dont know how this stay related with dxvk
